### ISSUE TITLE
Can Someone Build This?

### ISSUE THREAD

**Issue by AmazingMech2418** [GrooveProgramming/editor#17]
I just added a "Hello World" block. Can someone clone this repo, run build.py, and push the resulting code?

**Comment by jemunm378** [comment]
> I just added a "Hello World" block. Can someone clone this repo, run build.py, and push the resulting code? This is just for testing purposes. You don't even have to push the resulting code as long as you can check if the "Hello World" block appears.

Say that again?

**Comment by B294** [comment]
You want us to run the build?

**Comment by Spaceflyer234** [comment]
Once I get on my computer I will.

**Comment by AmazingMech2418** [comment]
@B294 Yes. Under the "text" section, there should be a "hello world" block once you build.

**Comment by jemunm378** [comment]
> @B294 Yes. Under the "text" section, there should be a "hello world" block once you build.

Can I fork it?

**Comment by AmazingMech2418** [comment]
@jemunm378 There is no need to fork it. All that is needed is to clone or download to get an offline version running and running build.py.

**Comment by AmazingMech2418** [comment]
Also, the reason why the sound blocks don't show is because it hasn't been built and the sound blocks are not in a proper js file.

**Comment by jemunm378** [comment]
> Also, the reason why the sound blocks don't show is because it hasn't been built and the sound blocks are not in a proper js file.

Oops, can you fix it?

**Comment by B294** [comment]
> > Also, the reason why the sound blocks don't show is because it hasn't been built and the sound blocks are not in a proper js file.
> 
> Oops, can you fix it?

Speaking of which... This is a block editor, not a paint editor.

**Comment by jemunm378** [comment]
> > > Also, the reason why the sound blocks don't show is because it hasn't been built and the sound blocks are not in a proper js file.
> > 
> > 
> > Oops, can you fix it?
> 
> Speaking of which... This is a block editor, not a paint editor.

What?

**Comment by B294** [comment]
> > > > Also, the reason why the sound blocks don't show is because it hasn't been built and the sound blocks are not in a proper js file.
> > > 
> > > 
> > > Oops, can you fix it?
> > 
> > 
> > Speaking of which... This is a block editor, not a paint editor.
> 
> What?

Lol, it turned into a paint editor last night.
